lib/miq_expression.rb

```diff
@@ -1124,7 +1124,7 @@ def self._custom_details_for(model, options)
 
     klass.custom_keys.each do |custom_key|
       custom_detail_column = [model, CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX + custom_key].join("-")
-      custom_detail_name = custom_key
+      custom_detail_name = "CustomAttribute: " + custom_key.capitalize
```

Comment:
Yes, if the `"Custom Attribute:"` string is to be propagated into UI, it needs to be inside gettext.